server/user-bootstrap/index.js

```diff
@@ -42,14 +61,24 @@ module.exports = function( authCookieValue, geoCountry ) {
 				);
 			}
 
-			hmac = crypto.createHmac( 'md5', API_KEY );
+			const hmac = crypto.createHmac( 'md5', API_KEY );
 			hmac.update( authCookieValue );
-			hash = hmac.digest( 'hex' );
+			const hash = hmac.digest( 'hex' );
 
 			req.set( 'X-Forwarded-GeoIP-Country-Code', geoCountry );
 			req.set( 'Authorization', 'X-WPCALYPSO ' + hash );
 			req.set( 'Cookie', AUTH_COOKIE_NAME + '=' + authCookieValue );
 			req.set( 'User-Agent', 'WordPress.com Calypso' );
+		} else if ( supportSession ) {
+			const hmac = crypto.createHmac( 'md5', SUPPORT_SESSION_API_KEY );
+			hmac.update( supportSession );
+			const hash = hmac.digest( 'hex' );
+			req.set( 'Authorization', `X-WPCALYPSO-SUPPORT-SESSION ${ hash }` );
+
+			setSupportSession( req, supportSession );
+		} else {
+			reject( new Error( 'Cannot bootstrap without an auth cookie or a support session.' ) );
+			return;
```

Comment:
I normally favor explicitly handling the `else` case for clarity and the hope that it is harder to accidentally break than handling the case elsewhere. But I'm not passionate about it here, especially since this is a nice early return and allows us to focus on a happier path. I'll make the change.